lib/commands/clipboard.js

```diff
@@ -0,0 +1,40 @@
+import { exec } from 'teen_process';
+
+const commands = {};
+
+/**
+ * Sets the clipboard's content on the Windows PC.
+ *
+ * @param {!string} content - The content to be set as base64 encoded string.
+ * @param {?string} contentType [plaintext] - The type of the content to set.
+ *                                            Only `plaintext` and 'image' are supported.
+ */
+commands.setClipboard = async function setClipboard (content, contentType) {
+  if (contentType === 'plaintext') {
+    return await exec('powershell', ['-command', `Set-Clipboard -Value '${contentType}'`]);
+  } else if (contentType === 'image') {
+    return await exec('powershell', ['-command', `$img=[Drawing.Bitmap]::FromStream([IO.MemoryStream][Convert]::FromBase64String(${content}));[System.Windows.Forms.Clipboard]::SetImage($img);$img.Dispose();`]);
+  }
+  throw new Error(`Could not set clipboard as ${contentType} is not a supported content type.`);
+};
+
+/**
+ * Sets the clipboard's content on the Windows PC.
+ *
+ * @param {?string} contentType [plaintext] - The type of the content to get.
+ *                                            Only `plaintext` and 'image' are supported.
+ *                                            Images must be in PNG format.
+ * @returns {string} The actual clipboard content encoded into base64 string.
+ * An empty string is returned if the clipboard contains no data.
+ */
+commands.getClipboard = async function getClipboard (contentType) {
+  if (contentType === 'plaintext') {
+    return await exec('powershell', ['-command', `Get-Clipboard`]);
```

Comment:
exec returns an object. You probably need to only get the stdout property

Reply:
Of course - sorry, can't debug easily as the Java-Client bindings don't cast to HasClipboard for WindowsDriver until this implemented.  Forgot that!

Reply:
it is also possible to set `isBuffer = true` property for exec. Then the stdout will be represented as Buffer object. You could simply call `stdout.toString('base64')` to make it to the command result

Reply:
Good point - I was initially doing this as I was considering how to handle the different contentTypes, and thought if a function was extracted for handling multiple types, etc., it would be better to make sure that what is returned from stdout was the same, and for image it would have to out b64, so was trying to ensure plaintext did the same.  Happy to implement this way, though!  Let me know!

Reply:
there is no need to perform any conversion in the driver code if your power shell command line already returns base64 encoded string